### PR TITLE
Make IgnoreDts configurable for M3U tuner

### PR DESCRIPTION
**Pull request by knackebrot**
**Changes**
- Added IgnoreDts to TunerHostInfo
- Added IgnoreDts checkbox to M3U tuner configuration page: https://github.com/jellyfin/jellyfin-web/pull/3696

**Issues**

Fixes https://github.com/jellyfin/jellyfin/issues/7267
Closes https://github.com/jellyfin/jellyfin/pull/7779 and https://github.com/jellyfin/jellyfin/pull/7776

### CODE REVIEW

File: MediaBrowser.Model/LiveTv/TunerHostInfo.cs [hunk 1]
```diff
@@ -31,5 +31,7 @@ public TunerHostInfo()
         public int TunerCount { get; set; }
 
         public string UserAgent { get; set; }
+
+        public bool IgnoreDts { get; set; } = true;
```
Comment (crobibero) [inline MediaBrowser.Model/LiveTv/TunerHostInfo.cs:35]: `AllowHWTranscoding` is set in the constructor, so might as well keep it consistent

Comment (knackebrot) [reply MediaBrowser.Model/LiveTv/TunerHostInfo.cs]: Fixed

**Comment by thornbill** [comment]
Given the nature of this change is to add a new feature (including what appears to be an API change) this might be better suited for merging to master?

**Review by crobibero — APPROVED** [review]

**Review by Bond-009 — APPROVED** [review]

**Comment by rebel24** [comment]
I've updated to the latest build and don't see this option ? Is this in the latest unstable build, if not, can I set this parameter manually ?

**Pull request merged**